netbox/templates/inc/filter_list.html

```diff
@@ -37,7 +37,7 @@
         </div>
       {% endif %}
     </div>
-    <div class="card-footer text-end d-print-none border-0">
```

Comment:
We may want to move these buttons outside of the card for consistency with other floating button groups. (Also, the background color is off.)

![screenshot](https://github.com/user-attachments/assets/bec2a70a-2e1b-43fe-884c-21c8ff5e751d)

Reply:
This is done